### PR TITLE
Issue organisation certificate

### PR DESCRIPTION
**Pull request by reinkrul**

### CODE REVIEW

**Comment by reinkrul** [comment]
> Am I correct to assume that, if this is merged, new events are generated with a certificate but old events (existing) are not migrated Yet. This would probably require events that are an update of a previous event?

When this is merged, newly registered organizations will have a X.509 certificate (issued by vendor CA) in JWK (encoded in "x5c" JWK field). Migration does not actually migrate anything, but just informs the operator when there are organizations without a certificate, and that it will be auto-migrated in the future. However, to be able to migrate anything (be it manual or automatic), we really need to be able to update events, so that would be the next feature to be realized, in my opinion.

TL;DR: Migration is to come when we can update events, this PR "just" adds a certificate to the organization JWK concept.
